src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -62,17 +68,17 @@ StatusNotifierItem::StatusNotifierItem(QString id, QObject *parent)
 
 StatusNotifierItem::~StatusNotifierItem()
 {
-    QDBusConnection::sessionBus().unregisterObject("/StatusNotifierItem");
-    QDBusConnection::sessionBus().unregisterService(mService);
-    QDBusConnection::sessionBus().disconnectFromBus(mService);
+    mSessionBus.unregisterObject("/StatusNotifierItem");
```

Comment:
I don't want to argue, but what is the rationale for that? Wouldn't that be an extra conversion? (if you meant it like this: `mSessionBus.unregisterObject(QLatin1String("/StatusNotifierItem"));`)